src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -1788,6 +1815,26 @@ fn new_graph_editor(app:&Application) -> GraphEditor {
 
 
 
+    // ========================
+    // === Scene Navigation ===
+    // ========================
+
+    frp::extend! { network
+
+        no_vis_selected <- out.some_visualisation_selected.gate_not(&out.some_visualisation_selected);
+        some_vis_selected <- out.some_visualisation_selected.gate(&out.some_visualisation_selected);
+
```

Comment:
use `on_true` and `on_false` (available after mmerging my PR: https://github.com/enso-org/ide/pull/973)